app/views/admin/lead_provider_users/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for @user, url: admin_lead_provider_users_path do |f| %>
+  <%= f.govuk_text_field(
+        :first_name,
+        label: { text: 'First name' })
+  %>
+  <%= f.govuk_text_field(
+        :last_name,
+        label: { text: 'Last name' })
+  %>
+  <%= f.govuk_email_field(
+        :email,
+        label: { text: 'Email' })
+  %>
+  <%= f.govuk_submit "Create" %>
+<% end %>
```

Comment:
Can we commonise it with the update form? The only difference I can see is the text on submit button.

Reply:
And the form submission path